internal/python/plan.go

```diff
@@ -261,6 +266,24 @@ func DetermineWsgi(ctx *pythonPlanContext) string {
 		return ""
 	}
 
+	if framework == types.PythonFrameworkSanic {
+		entryFile := DetermineEntry(ctx)
+
+		re := regexp.MustCompile(`(\w+)\s*=\s*Sanic\([^)]*\)`)
+		content, err := afero.ReadFile(src, entryFile)
+		if err != nil {
+			return ""
+		}
+
+		match := re.FindStringSubmatch(string(content))
+		if len(match) > 1 {
+			entryWithoutExt := strings.Replace(entryFile, ".py", "", 1)
```

Comment:
```suggestion
			entryWithoutExt := strings.TrimSuffix(entryFile, ".py")
```

TrimSuffix is more efficient than Replace.